packages/http-server/src/__tests__/header-params-validity.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { createLogger } from '@stoplight/prism-core';
+import { IHttpOperation, HttpParamStyles } from '@stoplight/types';
+import * as fastify from 'fastify';
+import { createServer } from '..';
+import { IPrismHttpServer } from '../types';
+
+const logger = createLogger('TEST', { enabled: false });
+
+function instantiatePrism2(operations: IHttpOperation[]) {
```

Comment:
Why the number `2`?

Reply:
Some copy&paste "magic" in my editor :) . Have removed this test now.